### PR TITLE
[CI] fix the expected metadata of the images

### PR DESCRIPTION
**Pull request by jstourac**
Makes CI check for params.env happy again.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: ci/check-params-env.sh [hunk 1]
```diff
@@ -69,6 +69,7 @@ function check_image_variable_matches_name_and_commitref() {
     case "${image_variable}" in
         odh-minimal-notebook-image-n)
             expected_name="odh-notebook-jupyter-minimal-ubi9-python-3.9"
+            # expected_commitref="2024a" or main
             expected_commitref="2023b"
```
Comment (atheo89) [inline ci/check-params-env.sh:73]: This shouldn't be 2024a? :thinking:

Comment (jstourac) [reply ci/check-params-env.sh:73]: these should be - as is stated in the comment, but right now the images that are used are still compiled with the 2023b - I though that this is an expected temporary state as this was for some time before. Or has the all images been updated already? Can check later for this if you want.

Comment (atheo89) [reply ci/check-params-env.sh:73]: Alright, should be as the comment indicates. There have been recent changes resulting in upside-down modifications to the params.env. Below is the manifest params.env file that we will adhere to until the next release.
https://github.com/opendatahub-io/notebooks/blob/main/manifests/base/params.env

Comment (jstourac) [reply ci/check-params-env.sh]: Thanks, didn't notice it has been updated in the meantime. I rebased this PR and updated. Validation now passes with the expected ref values.

Only exception is habana, where we still have a `main`. I suppose that this image hasn't been updated to 2024a yet - which is also reason that there is still just one version available, correct - https://github.com/opendatahub-io/notebooks/blob/main/manifests/base/params.env#L13 ?

Comment (atheo89) [reply ci/check-params-env.sh]: Yeah, for habana we don't have any updates

**Pull request merged**